### DISCUSSION TITLE
How does one build freeTDS for 32 bit linux?

### DISCUSSION THREAD

**Discussion by dev5620** [FreeTDS/freetds#483] [Q&A] [answered]
Can 32 bit linux freeTDS be built from the main (master) branch, a different branch, tag, or set of compile time defines?

Dev5620

**Comment by freddy77** [comment, marked as answer]
Yes, why not?
Maybe the issue is compiling 32 bit on a 64 bit machine. In this case it's
like a cross compilation. You need to call configure in a different way.
Easier to use a 32 bit VM or container
